### ISSUE TITLE
Shall I include Chocolatey Package Manager with CleanWin? (read comment)

### ISSUE THREAD

**Issue by pratyakshm** [pratyakshm/CleanWin#8]
[![](https://api.gh-polls.com/poll/01ES1G33ZGH28DNYT9CGJ4VNNG/Yes)](https://api.gh-polls.com/poll/01ES1G33ZGH28DNYT9CGJ4VNNG/Yes/vote)
[![](https://api.gh-polls.com/poll/01ES1G33ZGH28DNYT9CGJ4VNNG/No)](https://api.gh-polls.com/poll/01ES1G33ZGH28DNYT9CGJ4VNNG/No/vote)

**Note:** WinGet will always ship with CleanWin. I've conducted this poll because some users might prefer Chocolatey over WinGet since WinGet is still in active development and lacks a few features that Chocolatey has. However, since WinGet adequately fulfills my needs, I would not be removing it. If the poll turns out to be positive towards Chocolatey, it will co-exist in an optional manner with WinGet. 
TLDR: If Chocolatey turns out to be the preferred option, I'm not going remove WinGet. Both of them will co-exist in an optional manner.

My decision will be the final on this matter.

**Comment by pratyakshm** [comment]
this poll will be closed in the next 24 hours.
